PrivacyCookie/DemoBundleBannerFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\DemoBundle\PrivacyCookie;
+
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\URLAliasService;
+use eZ\Publish\API\Repository\ContentService;
+use Symfony\Component\Translation\Translator;
+
+interface DemoBundleBannerFactory
```

Comment:
Why do you declare a new BannerFactory interface ? You already have one in the privacy cookie bundle package: https://github.com/ezsystems/EzSystemsPrivacyCookieBundle/blob/master/Twig/PrivacyCookieTwigExtension.php#L29.